app/Jobs/CirrusSearch/ForceSearchIndex.php

```diff
@@ -11,7 +11,7 @@
  * 
  * Example:
  * 
- * php artisan job:dispatchNow CirrusSearch\\ForceSearchIndex id 1 0 1000
+ * php artisan job:dispatchSync CirrusSearch\\ForceSearchIndex id 1 0 1000
```

Comment:
I wonder if we want `dispatch` instead. If I understand it correctly `dispatchSync` will not use the existing queue setup, but somehow bypass this. Right now, I would think we want this to run together with the other jobs instead.

But maybe this is also something we should look into with https://phabricator.wikimedia.org/T342866

Reply:
I just realized this console command is completely separate from the framework anyway... https://github.com/mxl/laravel-job/blob/master/src/Commands/DispatchNow.php

But still I think you are right, `dispatch` is probably a better choice most of the time. I'll change the comments accordingly

Reply:
There are some more critical instances where I'd leave it though, MediawikiUpdate.php for example https://github.com/wbstack/api/blob/e59eda7de24f1baccbb7a02c285f684eebe93ecd/app/Jobs/MediawikiUpdate.php